MediaPushPlugin/src/main/java/io/antmedia/plugin/MediaPushPlugin.java

```diff
@@ -43,6 +44,16 @@ public Map<String, WebDriver> getDrivers() {
 		return drivers;
 	}
 
+	private static final String antMediaStateReady = "ready";
```

Comment:
Pleaes use Java static final naming conventions

Reply:
It's fixed